com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/app/discovery/ZigBeeNodeServiceDiscoverer.java

```diff
@@ -235,6 +235,7 @@ private void startDiscovery(Set<NodeDiscoveryTask> newTasks) {
                 // Create a new node to store the data from this update.
                 // We set the network address so that we can detect the change later if needed.
                 updatedNode = new ZigBeeNode(networkManager, node.getIeeeAddress(), node.getNetworkAddress());
+                updatedNode.setNodeState(node.getNodeState());
```

Comment:
That makes sense, I guess setting it to ONLINE is the right way which will also ensure that the `ZigBeeDiscoveryExtension` doesn't stop the discovery because the `updatedNode` is in `UNKNOWN` state.
Should I go ahead and update the existing PR?

Reply:
> Should I go ahead and update the existing PR?


Yes, thanks.